ci/redmine_bugzilla.py

```diff
@@ -149,12 +150,22 @@ def main():
                         links_issues_record += 'Redmine #%s -> Bugzilla %s, but Bugzilla %s does not ' \
                                            'link back\n' % (issue.id, bug.id, bug.id)
 
+    start_time = time.time()
+    redmine_issues_looked_up = 0
     for bug in bugzilla_bugs:
         for external_bug in bug.external_bugs:
                 if external_bug['type']['description'] == 'Pulp Redmine':
                     add_cc_list_to_bugzilla_bug(bug)
                     issue_id = external_bug['ext_bz_bug_id']
-                    issue = redmine.issue.get(issue_id)
+                    try:
+                        redmine_issues_looked_up += 1
+                        issue = redmine.issue.get(issue_id)
+                    except:
+                        end_time = time.time()
+                        time_to_query_before_failure = start_time - end_time
+                        print 'Total time to query %d redmine issues: %f' %\
```

Comment:
Just saw this, and it's not entirely true:
```
$ python2.7
Python 2.7.12 (default, Nov  7 2016, 11:55:55) 
[GCC 6.2.1 20160830] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print('Hi')
Hi
```

This example explains why...
```
>>> print('Hello' 'World')
HelloWorld
>>> print('Hello', 'World')
('Hello', 'World')
```
The whitespace after `print` doesn't matter, so as long as you aren't using commas to turn what's inside the parenthesis into a tuple, the print statement will handle it just fine.

So as long as you don't get commas involved and turn this into a tuple, I think you're free to `print('foo')` in py2.7.